### PR TITLE
Improve Mux Player logger + Media Chrome bugfix for fullscreen

### PR DESCRIPTION
**Pull request by dylanjha**
✅ Improve Mux Player logger
✅ Media Chrome bugfix for fullscreen

Thoughts on improving the logger to include the player version number?

log_2022-05-12_09-55-20

### CODE REVIEW

**Review by cjpillsbury — APPROVED** [review]

File: packages/mux-player/src/html.ts [hunk 1]
```diff
@@ -1,12 +1,7 @@
 // NOTE: In @github/template-parts@0.5.2 the published package has some incorrect pathing for default modules.
```
Comment (gkatsev) [inline packages/mux-player/src/html.ts:1]: this comment isn't needed anymore.

**Review by gkatsev — CHANGES_REQUESTED** [review]

File: packages/mux-player/src/html.ts [hunk 2]
```diff
@@ -1,13 +1,5 @@
-// NOTE: In @github/template-parts@0.5.2 the published package has some incorrect pathing for default modules.
-// Pointing directly to the current deep (likely unintentionally published) paths until this is resolved in a later published version (CJP)
-import {
-  TemplateInstance,
-  NodeTemplatePart,
-  createProcessor,
-  AttributeTemplatePart,
-} from '@github/template-parts/lib/src/index.js';
-import type { TemplatePart, TemplateTypeInit } from '@github/template-parts/lib/src/types';
-// NOTE: These are either direct ports or significantly based off of github's jtml template part processing logic. For more, see: https://github.com/github/jtml
```
Comment (gkatsev) [inline packages/mux-player/src/html.ts:10]: do we want to keep this comment?

**Review by gkatsev — APPROVED** [review]

File: packages/mux-player/src/html.ts (hunk 2 above)
Comment (dylanjha) [reply packages/mux-player/src/html.ts:10]: I thought that was old from before we used template-parts

Comment (dylanjha) [reply packages/mux-player/src/html.ts:10]: re-visiting and looking at jtml, I think I was wrong though

**Pull request merged**